sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -52,8 +58,20 @@
     private final AzureKeyCredential credential;
     private final boolean sessionCapturingOverride;
     private final boolean enableTransportClientSharing;
+    private final TracerProvider tracerProvider;
     private final DataEncryptionKeyProvider dataEncryptionKeyProvider;
     private final boolean contentResponseOnWriteEnabled;
+    private static final Tracer TRACER;
+
+    static {
+        ServiceLoader<Tracer> serviceLoader = ServiceLoader.load(Tracer.class);
+        Iterator<?> iterator = serviceLoader.iterator();
+        if (iterator.hasNext()) {
+            TRACER = serviceLoader.iterator().next();
```

Comment:
what happens if there are multiple implementation? is that possible?

Reply:
There can be multiple implementations in the classpath. But there is no real use case to support multiple implementations as the user will always tie a specific implementation to an exporter.